pkg/awsvpc/explainability_test.go

```diff
@@ -23,6 +24,33 @@ var explainTests = []*commonvpc.VpcGeneralTest{
 		Format:        vpcmodel.Text,
 		DetailExplain: true,
 	},
+	// connection to the public internet blocked by sg and private subnet
+	{
+		Name:          "to_external_private_subnet",
+		InputConfig:   "aws_sg_1",
+		ESrc:          "10.240.20.245",
+		EDst:          "161.26.0.0",
+		Format:        vpcmodel.Text,
+		DetailExplain: true,
+	},
+	// existing connection to the public internet
+	{
+		Name:          "to_external_public_subnet",
+		InputConfig:   "aws_sg_1",
+		ESrc:          "10.240.10.42",
+		EDst:          "161.26.0.0",
+		Format:        vpcmodel.Text,
+		DetailExplain: true,
+	},
+	// connection to the public internet blocked only by private subnet
+	{
+		Name:          "to_external_blocked_only_private_subnet",
+		InputConfig:   "aws_sg_1_allow_sg_private",
```

Comment:
config files should be added to the tests only from the teraform example repository.
and should not be edited.
if you need an specific example, please ask Karen
also notice that we also have aws_acl_1 and aws_mixed examples, to use

Reply:
We have added manual modified examples in the past. 
However, I opened an issue https://github.com/np-guard/vpc-network-config-analyzer/issues/846 for Karen. Once she generate the required example I will delete mine.
In the meantime lets not have it a bottleneck (Karen said she'll probably get to it Sunday), and have this PR merged.